### DISCUSSION TITLE
[Feature] <optional uploaded PNG convert to jxl lossless>

### DISCUSSION THREAD

**Discussion by chnhi** [immich-app/immich#5118] [Feature Request]
### The feature

as chrome dissupport jxl  ,lossy formart convert to jxl is unnessary.
but png is a lossless formart ,switch to a more modern formart may be useful.
use avif as thumbnali which support alpha channel

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by mmomjian** [comment]
We do not plan on ever editing the original assets.
